baker/SiteBaker.tsx

```diff
@@ -779,6 +779,25 @@ export class SiteBaker {
         this.progressBar.tick({ name: "✅ baked google drive images" })
     }
 
+    // Make sure that our tag icons don't have typos, nor reference tags that have been renamed
+    // We want to be sure that if we have a tag name, we'll be able to load its icon like so:
```

Comment:
nit: this confused me a tiny bit. For me, this implies that this function makes sure _every_ tag name gets its icon (when in reality, it's the other way around)